### DISCUSSION TITLE
How to share same instance across multiple trees?

### DISCUSSION THREAD

**Discussion by ParadiseFallen** [mobxjs/mobx-state-tree#1980] [Q&A]
Hi/ i have kinda `Window` store (window that can be closed and dragged).

So i has `Sidenav` and `Messages` models. (ViewModels thing). So i need to pass Window to Sidenav and messages but got error 
`
Uncaught Error: [mobx-state-tree] Cannot add an object to a state tree if it is already part of the same or another state tree. Tried to assign an object to '/window', but it lives already at '/window'`

**Comment by EmilTholin** [comment]
Hi @ParadiseFallen!

Do you think you could include a minimal, reproducible example in e.g. a [CodeSandbox](https://codesandbox.io/s/)? It's difficult to say what might go wrong without seeing some code.
